config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml

```diff
@@ -184,6 +184,55 @@ periodics:
       secret:
         secretName: k8s-triage-robot-github-token
 
+- name: ci-k8s-triage-robot-reopen
```

Comment:
We initially wanted to avoid closing those issues, but I wasn't able to write a query that would achieve that.

So I switched to simply reopening the ones that shouldn't be close - that has an additional advantage of reopening the ones that were already closed and shouldn't be.

Reply:
hmmm... is dueling bots really what we want? won't the lifecycle bot just close it again?

Reply:
it won't because the issue was touched (and it closes the ones not touched for 720h).

Anyway - if you know how to do a query that will exclude the isues we want to exclude - I'm happy to change. I just don't know how if it's even possible in github query syntax

Reply:
This is fundamentally different than "don't close issues matching X".. won't this reopen issues that are already actually fixed too? This will also attempt to reopen PRs.

Reply:
> This will also attempt to reopen PRs.

That's actually advantage - it will reopen those that we shouldn't have closed.
Currently there are 6 such closed issues - so not a big change.

Re fixed issue - hopefully people remove the "lifecycle rotten" label when they start working on things - the above number seems to confirm that.

Reply:
Updated based on slack conversation.